### DISCUSSION TITLE
Black and white mode is gone?

### DISCUSSION THREAD

**Discussion by minh0722** [FarGroup/FarManager#721] [Q&A] [answered]
Hi, the last version I used which is 3.0.5700, the black and white mode is available, now it's gone. How can I set it in the latest version?
Thanks

**Comment by alabuzhev** [comment, marked as answer]
@minh0722 it has been moved to where all the other color schemes are.
You can enable it with `far /import %farhome%\Addons\Colors\Interface\black_and_white.farconfig`

**Reply by minh0722** [reply]
great, thanks!
